src/worker/exporters/base_exporter.py

```diff
@@ -29,9 +29,9 @@ def init_connection(self):
 
     def init_gauges(self):
         '''Initialization of Prometheus parameters. Override in Child Class if needed'''      
-        self.guages = {}
+        self.gauges = {}
```

Comment:
I don't see any changes in this file? what was changed? If nothing was can you revert this file back to the original.

Reply:
Gauges was misspelled, which caused problems in the child class NodeExporter